pkgs/servers/metabase/default.nix

```diff
@@ -0,0 +1,28 @@
+{ pkgs, stdenv, fetchurl, makeWrapper, jre }:
+
+stdenv.mkDerivation rec {
+  name = "metabase-${version}";
+  version = "0.28.1";
+
+  jar = fetchurl {
+    url = "http://downloads.metabase.com/v${version}/metabase.jar";
+    sha256 = "1nv3y4pnvzd7lwyj14nmhr3k52qd8hilcjxvd7qr3hb7kzmjvbzk";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  unpackPhase = "true";
+
+  installPhase = ''
+    mkdir -p $out/share/java
+    ln -s $jar $out/share/java/metabase.jar
+    makeWrapper ${jre}/bin/java $out/bin/metabase --add-flags "-jar $out/share/java/metabase.jar"
```

Comment:
Not important, but I wonder if creating the dir/symlink is necessary; what if you just pass `-jar $jar`?
In that case, the whole expression reduces to, basically
```nix
writeScriptBin "metabase" ''
  #! /bin/sh
  exec ${jre}/bin/java -jar ${jar} "$@"
''
```
Just something to consider.

Reply:
It is probably not necessary, I am only not very Nix experienced yet.
How does `writeScriptBin` work?
I tried
```nix
  installPhase = writeScriptBin "metabase" ''
    #!/bin/sh
    exec ${jre}/bin/java -jar ${jar} "$@"
  '';
```
But it fails with `/nix/store/…/*-metabase: Is a directory`. I grepped for usage examples but could only find packages where `writeScriptBin` is used in the `let` scope for dependencies.

Reply:
`writeScriptBin` describes a derivation that given a name and text produces an output comprising an executable script of that name containing that text under a `bin/` directory.

In your case, `installPhase` is supposed to be a Nix string  (which is then exported to the builder); you're instead setting it to be a Nix derivation (evaluating to a store path). 

What I was suggesting was that the whole package expression would reduce, essentially, to `writeScriptBin`. That is, no `installPhase` or anything. I'm not suggesting you do literally that :)

In the context of your original expression, something like (untested)
```
installPhase = ''
  mkdir -p $out/bin
  cat >$out/bin/metabase <<EOF
  #! /bin/sh
  exec ${jre}/bin/java -jar ${jar} "$@"
  EOF
''
```
might work.

Anyway, this is not at all a blocker. Please just remove the redundant `pkgs` parameter and I'd be happy to merge as-is.

Reply:
Thanks a lot for your explanation! :heart: I decided to stick with the verbose approach because it is more common.

> Not important, but I wonder if creating the dir/symlink is necessary; what if you just pass `-jar $jar`?

Fixed.